components/Input/DateFilter/index.js

```diff
@@ -1,54 +1,107 @@
 import PropTypes from 'prop-types';
-import React, { Fragment } from 'react';
+import React from 'react';
+import memoize from 'memoize-one';
 
+import { requiredCondition } from '../../General/Faram';
 import { FaramInputElement } from '../../General/FaramElements';
+import FormattedDate from '../../View/FormattedDate/FormattedDate';
+import ApplyModal from '../../View/ApplyModal';
+
+import { encodeDate, decodeDate } from '../../../utils/common';
 
 import SelectInput from '../SelectInput';
 import DateInput from '../DateInput';
-import PrimaryButton from '../../Action/Button/PrimaryButton';
-import DangerButton from '../../Action/Button/DangerButton';
-import Modal from '../../View/Modal';
-import ModalHeader from '../../View/Modal/Header';
-import ModalBody from '../../View/Modal/Body';
-import ModalFooter from '../../View/Modal/Footer';
-
-import { formatDate } from '../../../utils/date';
-import { encodeDate } from '../../../utils/common';
 
 import styles from './styles.scss';
 
+const noOp = () => {};
 
 const propTypes = {
-    /**
-     * for styling
-     */
     className: PropTypes.string,
-
-    onChange: PropTypes.func,
-
-    /**
-     * Placeholder for the input
-     */
-    placeholder: PropTypes.string,
-
     value: PropTypes.shape({
-        type: PropTypes.string,
-        startDate: PropTypes.oneOfType([
-            PropTypes.number,
-            PropTypes.string,
-        ]),
-        endDate: PropTypes.oneOfType([
-            PropTypes.number,
-            PropTypes.string,
-        ]),
+        startDate: PropTypes.string,
+        endDate: PropTypes.string,
     }),
+    onChange: PropTypes.func,
 };
 
 const defaultProps = {
     className: '',
-    placeholder: 'Select an option',
-    value: undefined,
-    onChange: undefined,
+    value: {},
+    onChange: noOp,
+};
+
+const formatDate = date => FormattedDate.format(decodeDate(date), 'dd-MM-yyyy');
```

Comment:
OMG, FormattedDate.format is not there anymore. 404

Reply:
OMG, who moved this without notifying others or keeping backward compatibility. This is 410: resource intentionally is gone.